README.md

```diff
@@ -43,15 +58,29 @@ import com.banuba.agora.plugin.model.ArEffect
 import com.banuba.sdk.utils.ContextProvider
 ```
 
+Create the Banuba SDK extension object:
+
+```kotlin
+private val banubaExtension = BanubaExtensionManager(object : AgoraInterface {
+  override fun onSetExtensionProperty(provider: String, extension: String, propertyKey: String, propertyValue: String) {
+    agoraRtc.setExtensionProperty(provider, extension, propertyKey, propertyValue)
+  }
+})
+
+/* This interface separates banuba SDK and Agora rtc engine from each other.
+* In this example, it is only demonstrative, but it can be useful in large projects. */
```

Comment:
Translation via google: "В данном примере это демонстративно, но может быть полезно в больших проектах."
In this example, it is only for demonstration, but it can be useful in large projects. - "В данном примере это только для демонстрации, но может быть полезно в больших проектах."

Again, the second variant is clearer.